ui/src/Contexts/README.md

```diff
@@ -0,0 +1,38 @@
+# React Contexts
```

Comment:
Might be worth talking about state management at a higher level rather than just in the contexts folder? I think it'll bee easier to come across then as it's something important.

Can do!

Reply:
Agreed - happy to put something together re state (as you are adding the first of it?)

Reply:
Yep! Already added into `ui/docs/CodeStyle.md`